package.json

```diff
@@ -105,7 +105,8 @@
     "rollup-plugin-url": "^2.1.0",
     "semver": "^7.1.3",
     "shelljs": "^0.8.3",
-    "webpack": "^4.41.0"
+    "webpack": "^4.41.0",
+    "worker-plugin": "^4.0.3"
```

Comment:
There is no way to handle this with regular `Worker` or `rollup` based solution? we do not use `webpack` here

Reply:
unfortunately, the worker loader works with the Webpack file loader, i'm gonna check the comlink also, to see if it needs the Webpack file loader or not, otherwise, it would be the only way

Reply:
as I checked the history of the package.json, the webpack had been added it the first commit!